web/beacon-app/src/features/projects/schemas/createProjectTopicSchema.ts

```diff
@@ -9,7 +9,10 @@ export const createProjectTopicSchema = Yup.object().shape({
     .trim()
     .required(t`Topic name is required.`)
     .matches(/^[^\s]*$/, t`Topic name cannot include spaces.`)
-    .matches(/^[^_-].*$/, t`Topic name cannot start with an underscore or dash.`)
+    .matches(
+      /^[a-zA-Z][a-zA-Z0-9._-]*$/,
+      t`Topic name may only start with a letter and contain letters, numbers, underscores, and dashes.`
```

Comment:
I know we want to provide users with helpful info on why their topic name is not accepted. If you think this should be changed, let me know.